openapi/ogcapi-processes.bundled.json

```diff
@@ -136,6 +136,54 @@
             "$ref": "#/components/responses/ProcessList"
           }
         }
+      },
+      "post": {
+        "summary": "deploy a process.",
+        "description": "Deploys a process.\n\nFor more information, see [Section 6.3](http://docs.ogc.org/DRAFTS/20-044.html#_87a6983e-d060-458c-95ab-27e232e64822).\n",
+        "operationId": "deploy",
+        "tags": [
+          "DRU"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/components/parameters/w"
+          }
+        ],
+        "requestBody": {
+          "description": "An OGC Application Package used to deploy a new process.",
+          "required": true,
+          "content": {
+            "application/ogcapppkg+json": {
+              "schema": {
+                "$ref": "#/components/schemas/ogcapppkg"
+              }
+            },
+            "application/cwl": {
+              "schema": {
+                "type": "string"
+              }
+            },
+            "application/cwl+json": {
+              "schema": {
+                "example": {}
+              }
+            }
```

Comment:
Missing the latest updates from https://github.com/opengeospatial/ogcapi-processes/blob/7f3ab3c570b7fa71c977fbbe6a7500e3c78ff7ad/openapi/paths/processes-dru/pDeploy.yaml#L20-L28